casr/Cargo.toml

```diff
@@ -11,6 +11,8 @@ license = "Apache-2.0"
 exclude = ["/tests"]
 
 [dependencies]
+lettre = "0.11.15"
+secrecy = "0.8"
```

Comment:
Please, make these dependencies available only when casr is built with `dojo` feature (similar to tokio crate).